src/credentials/credentialsUtilities.ts

```diff
@@ -46,3 +47,7 @@ export function notifyUserInvalidCredentials(credentialProviderId: CredentialsPr
             }
         })
 }
+
+export function hasProfileProperty(profile: Profile, propertyName: string): boolean {
+    return !!profile[propertyName]
```

Comment:
Nit: if this won't ever have `profile[propertyName] === ''`, it's safer to do `return profile.hasOwnProperty(propertyName)` just in case `profile[propertyName]` is `false`, `0`, or some other falsy value, that being said TS should enforce this as a string so you should be fine (checked that string representations of some common falsy values appear to be fine with this).